### PR TITLE
yaml documentation

### PR DESCRIPTION
**Pull request by simpslandyy**
Added documentation for applicants, application, and assignment endpoints

### CODE REVIEW

**Review by freeatnet — APPROVED** [review]

File: docs/api/applicants.yml [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+        required: true
+        type: "integer"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/Assignee"
+        404:
+          description: "invalid ID supplied"
+  /applicants/{id}/assignment:
```
Comment (michellemtchai) [inline docs/api/applicants.yml]: This line should be deleted. It causes errors when it tries to compiles on Swagger. Also, I think you meant `../assignments` not `../assignment`.

File: docs/api/applicants.yml [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+        description: "Number of hours to update assignee"
+        required: true
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/Assignee"
+        404:
+          description: "applicant id or assignment id not found"
+  /applicants/{id}/assignments/{assignmentid}:
```
Comment (michellemtchai) [inline docs/api/applicants.yml]: The duplication of `/applicants/{id}/assignments/{assignmentid}:` causes a compile error on Swagger. You should delete this line.

File: docs/api/applications.yml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/Application"
+        404:
+          description: "invalid ID supplied"
+definitions:
+  Applications:
+    type: "object"
+      properties:
```
Comment (michellemtchai) [inline docs/api/applications.yml]: There's an indentation error for everything under (and including) `properties`. They should all be one tab backwards. The indentation issue creates a compilation error on Swagger.

File: docs/api/assignments.yml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            description: "successful operation"
+            schema:
+              $ref: "#/definitions/Assignment"
+          404:
+            description: "invalid ID supplied"
+
+definitions:
+  Assignment:
+    type: "object"
+      properties:
```
Comment (michellemtchai) [inline docs/api/assignments.yml]: There is also an indentation error on this `properties` attribute.

**Comment by freeatnet** [comment]
Can this be merged? @simpslandyy @michellemtchai

**Comment by michellemtchai** [comment]
<s>@freeatnet I think</s> @simpslandyy <s>needs to fix this before merging.</s> The YML file are not compiling.

@michellemtchai - you need to direct your concerns with me not with Arseniy. He can't do anything about me fixing them but tell me to fix them.
- Landry

@simpslandyy Sorry, I didn't realize I typed in the wrong name. 
-Michelle

**Review by michellemtchai — APPROVED** [review]

**Pull request merged**